backend/app/models/licenses.py

```diff
@@ -10,12 +10,11 @@
 
 class LicenseBase(BaseModel):
     name: str
-    description: str
-    url: str
-    version: str
-    holders: str
-    expiration_date: datetime = Field(None)
-    dataset_id: PydanticObjectId
+    description: str = None
+    url: str = None
+    version: Optional[str] = None
+    holders: Optional[str] = None
+    expiration_date: Optional[datetime] = Field(None)
```

Comment:
Any reason you need Field here? According to pydantic it's meant for adding metadata to field
https://docs.pydantic.dev/latest/concepts/fields/